### PR TITLE
EMSUSD-1555 Duplicating a Physical Sky to USD using Arnold options is missing data

### PR DESCRIPTION
**Pull request by barbalt**
When "Duplicate as Usd data" is select from the context ops menu on a maya object, we only translate to Usd the selected maya object.
This can be a problem when there are extra prims created by a chaser, for example. Those would be discarded.

With those changes, we allow the users to register the path for those extra prims created so that it can be queried when merging the maya object into Usd.

### CODE REVIEW

**Review by pierrebai-adsk — DISMISSED** [review]
Great!

**Comment by pierrebai-adsk** [comment]
IDK if we would not be better off squashing all those commits into a single one? Maybe creating a new PR?

**Comment by barbalt** [comment]
> IDK if we would not be better off squashing all those commits into a single one? Maybe creating a new PR?
@pierrebai-adsk I can do that. Out of curiosity, can't we squash during merge?

File: lib/mayaUsd/fileio/jobs/writeJob.cpp [hunk 1]
```diff
@@ -640,12 +634,19 @@ bool UsdMaya_WriteJob::_FinishWriting()
...
+
     // Run post export function on the chasers.
     MayaUsd::ProgressBarLoopScope chasersLoop(mChasers.size());
     for (const UsdMayaExportChaserRefPtr& chaser : mChasers) {
         if (!chaser->PostExport()) {
             return false;
         }
+
+        // Collect extra prims paths from chasers
+        for (const SdfPath& path : chaser->GetExtraPrimsPaths()) {
```
Comment (pierrebai-adsk) [inline lib/mayaUsd/fileio/jobs/writeJob.cpp:647]: I think there is a way to insert a range in C++ std::vector? Not super important.

**Review by pierrebai-adsk — APPROVED** [review]

**Pull request closed without merging**